src/App.css

```diff
@@ -1 +1,21 @@
 /* not empty */
+.pagination {
+  width: 20%;
+}
+
+.page-item{
+  padding-inline: 10px;
+  border: 1px solid grey;
+}
+
+.page-item.active {
+  background: blue;
+}
+
+.active .page-link{
+  color: white;
+}
+
+li.disabled a[aria-disabled="true"] {
```

Comment:
The selector for disabled links is good in terms of accessibility. However, make sure that the `li` element has a class that you can use to increase specificity, similar to the `.page-item` comment. For example, if `li` elements have a `.page-item` class, the selector could be `.page-item.disabled a[aria-disabled="true"]`.